src/gc.c

```diff
@@ -2957,7 +2957,7 @@ static int _jl_gc_collect(jl_ptls_t ptls, jl_gc_collection_t collection)
     uint64_t start_mark_time = jl_hrtime();
     JL_PROBE_GC_MARK_BEGIN();
     {
-        JL_TIMING(GC, Mark);
```

Comment:
@topolarity Is there a way to attach a message? Would be helpful to differentiate between minor and major GC

Reply:
In Tracy, we distinguish full sweeps based on the color associated with the event. That's one of the cases we haven't pulled up to a generic API yet: https://github.com/JuliaLang/julia/blob/552dd4c51c537e838d83548e044800a69d240f9a/src/gc.c#L3122-L3127

`jl_timing_printf` and `jl_timing_show_*` also allow associating strings with events, so we could use that if ITTAPI supports string metadata. Or did you have something more specialized in mind?

Reply:
I feel like if we had a conditional event that would be sufficient here.